### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by liuxing007** [Walter0807/MotionBERT#72]
Great work! I have a few questions:

(1) In the second version, the MB_train_h36m.yaml file was modified to set dim_feat as 512, whereas the paper mentions it as 256. Will this change improve the results?

(2) Without making any modifications, I ran the latest code for training 3D human pose estimation from scratch. However, the model file size in the checkpoint is 486.2MB, which is different from the 162MB model file uploaded by your team. Nevertheless, I am able to evaluate your model file and our trained 486.2MB model file directly. I'm curious to know the reason behind this.

(3) How many epochs did you train the model for? I have been unable to achieve the results mentioned in the paper. The best reported MPJPE is 39.4mm.

**Comment by Walter0807** [comment]
1. It is the main difference between MB and MB_lite. The paper version is also 512 (updated).
2. The saved checkpoint also contains training parameters (gradients, optimizer states, etc.), which are not needed for inference, and therefore we removed them in the release.
3. The number of training epochs is in the config. Did you use the same batch size and learning rate?

**Comment by liuxing007** [comment]
![QQ截图20230720105757](https://github.com/Walter0807/MotionBERT/assets/129535338/aa10285c-6210-4152-83a5-2e5d7359e6af)
This is the result of training with dim_feat=256. The best reported MPJPE is 39.9mm. batch size and learning rate are the same.

**Comment by liuxing007** [comment]
![图片](https://github.com/Walter0807/MotionBERT/assets/129535338/2a174f44-031a-4165-ae0a-7662a01ef4ad)
 This is the result of training with dim_feat=512. The best reported MPJPE is 39.4mm. batch size and learning rate are the same.
